HAL/pico/include/config_defaults.hpp

```diff
@@ -7,9 +7,9 @@
 // clang-format off
 
 const Config default_config = {
-    .game_mode_configs_count = 6,
+    .game_mode_configs_count = 7, // originally was 6, +1 for Rivals 2 Mode
     .game_mode_configs = {
-        GameModeConfig {
+    GameModeConfig { //even with all GameModeConfigs commented out besides one, the code works the same and switches modes the same
```

Comment:
Sorry I missed this the first time but could you remove these comments as well as they don't really make sense to leave there outside the context of this PR